README.md

```diff
@@ -167,6 +167,32 @@ $bunny = new Client($configuration);
 $bunny->connect();
 ```
 
+### Always run moving parts in a fiber
+
+The major different between Bunny v0.5 and v0.6 is that it's been rebuild using [`Fibers`](https://reactphp.org/async/). This means that every method on `Client` and `Channel` must be called inside a fiber. In the examples this will be called using a `Loop::futureTick` call. In your applications this might be some other trigger like a HTTP request. As long as somewhere in the direct call stack this call is inside a fiber you'll be good.
```

Comment:
IMO I think part of "from 0.5 to 0.6" is necessary for the upgrade migration path, but not a generic chapter in the readme

```suggestion
Since v0.6 Bunny has been rebuilt using [`Fibers`](https://reactphp.org/async/). This means that every method on `Client` and `Channel` must be called inside a fiber. In the examples this will be called using a `Loop::futureTick` call. In your applications this might be some other trigger like a HTTP request. As long as somewhere in the direct call stack this call is inside a fiber you'll be good.
```

Reply:
You're right, it makes more sense in the release notes than here. Thanks for the rewording 👍